src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -745,6 +749,59 @@ export class PatientViewPageStore {
         invoke: ()=>Promise.resolve(indexHotspotsData(this.hotspotData))
     });
 
+    readonly sampleToMutationnGenePanelData = remoteData<{[sampleId: string]: GenePanelData}>({
```

Comment:
`Mutation`(one n)

Reply:
I corrected the name. Well spotted.

Reply:
it's looking the same right now, I guess you have it changed on local?

Reply:
Indeed, change was pushed and now it shows.